clang-tidy-required-packages.yaml

```diff
@@ -0,0 +1,223 @@
+##### common #####
```

Comment:
I felt that it might be a little difficult to manage this file when the package name changes or when packages are deleted/added.

Reply:
I agree.
However, this file is temporary. In the near future, all packages will pass the clang-tidy check and this file will not be needed. Ideally, we would like to make all packages clang-tidy check and make this check required right now, but that is unrealistic. This file setting is a temporary alternative and will be deleted.